### PR TITLE
[Task 1871990] move ISecureStore to AzureStoreManager to prepare the SecureStore in eclipse

### PR DESCRIPTION
**Pull request by andxu**
[AB#1871990](https://dev.azure.com/mseng/a4d27ce2-a42d-4b71-8eef-78cee9a9728e/_workitems/edit/1871990)

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/FileStore.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import com.microsoft.azure.toolkit.ide.common.store.IFileStore;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class FileStore implements IFileStore {
+
+    @Nullable
+    @Override
+    public String getProperty(@Nonnull String s, @Nonnull String s1) {
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/FileStore.java]: Please also update the parameter name as well

File: Utils/azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/java/com/microsoft/azure/toolkit/ide/common/store/AzureStoreManager.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    private IApplicationStore appStore;
+
+    @Getter
+    private ISecureStore secureStore;
+
+    public static synchronized void register(@Nonnull IFileStore fileStore,
+                                             @Nonnull IApplicationStore appStore,
+                                             @Nonnull ISecureStore secureStore) {
+        getInstance();
+        instance.fileStore = fileStore;
```
Comment (Flanker32) [inline Utils/azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/java/com/microsoft/azure/toolkit/ide/common/store/AzureStoreManager.java]: fileStore -> machineStore?

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/FileStore.java (hunk 1 above)
Comment (andxu) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/FileStore.java]: Will impl it in later PR

File: Utils/azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/java/com/microsoft/azure/toolkit/ide/common/store/AzureStoreManager.java (hunk 2 above)
Comment (andxu) [reply Utils/azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/java/com/microsoft/azure/toolkit/ide/common/store/AzureStoreManager.java]: will rename it in later PR

**Review by Flanker32 — APPROVED** [review]
LGTM

**Pull request merged**